tests/python/unit/test_version.py

```diff
@@ -0,0 +1,29 @@
+"""Tests for version access
+
+Ensures that `compass.__version__` is exposed and semantically shaped.
+We allow dev/local forms produced by setuptools_scm
+(e.g., 0.11.3.dev8+gHASH).
+"""
+
+import re
+
+import compass
+
+
+SEMVER_DEV_PATTERN = re.compile(r"^\d+\.\d+\.\d+(?:\.dev\d+\+g[0-9a-f]+)?$")
```

Comment:
We haven't been using, but should we allow for release candidates (`rc`) as well? I think we will start using that in the future.

I don't have strong opinion about this. We can just wait

Reply:
I would use `rc` for the final checks. Share the `rc` wheels with the team and maybe use it for some short time to confirm before releasing so we would give more resilience for everyone using `compass`. That said, it would be just fine to use only `dev`, so let's keep only `dev` for now.